src/coreclr/jit/compiler.cpp

```diff
@@ -4151,14 +4151,11 @@ void Compiler::compSetOptimizationLevel()
             codeGen->setFrameRequired(true);
         }
 
-#if !defined(TARGET_AMD64)
         // The VM sets JitFlags::JIT_FLAG_FRAMED for two reasons: (1) the DOTNET_JitFramed variable is set, or
         // (2) the function is marked "noinline". The reason for #2 is that people mark functions
-        // noinline to ensure the show up on in a stack walk. But for AMD64, we don't need a frame
```

Comment:
The deleted comment is still true on Windows. I guess it was written before we cared about non-Windows.